include/DetuningHelper.h

```diff
@@ -43,8 +43,7 @@ class DetuningHelper : public InlineAutomation
 	}
 
 	~DetuningHelper() override
-	{
-	}
+	= default;
```

Comment:
Many lines contain only `= default` now. Grep those and move them behind the actual function?

Reply:
Done.